nautobot_golden_config/nornir_plays/config_compliance.py

```diff
@@ -130,8 +131,13 @@ def config_compliance(job_result, data):
     now = datetime.now()
     rules = get_rules()
     logger = NornirLogger(__name__, job_result, data.get("debug"))
-    global_settings = GoldenConfigSetting.objects.first()
-    verify_global_settings(logger, global_settings, ["backup_path_template", "intended_path_template"])
+
+    qs = get_job_filter(data)
+    device_to_settings_map = get_device_to_settings_map(queryset=qs)
+
+    for settings in set(device_to_settings_map.values()):
```

Comment:
using `set` here we skip verifying settings that share the config?

Reply:
We only skip the settings that do not apply for the job execution (ie. 10/1000 devices affected by the job -> 1/10 settings only applicable to these 10 devices.)